src/core/qgsbrowserproxymodel.cpp

```diff
@@ -122,19 +133,19 @@ bool QgsBrowserProxyModel::filterAcceptsString( const QString &value ) const
     case Normal:
     case Wildcards:
     {
-      for ( const QRegExp &rx : mREList )
+      for ( const QRegularExpression &rx : mREList )
       {
-        if ( rx.exactMatch( value ) )
+        if ( rx.match( value ).hasMatch() )
           return true;
       }
       break;
     }
 
     case RegularExpression:
     {
-      for ( const QRegExp &rx : mREList )
+      for ( const QRegularExpression &rx : mREList )
       {
-        if ( rx.indexIn( value ) != -1 )
+        if ( rx.match( value ).hasMatch() )
```

Comment:
these weren't exact matches -- so the use of anchored pattern above isn't correct in this case

Reply:
This is still outstanding